### PR TITLE
WoE: Space Invaders should always work, no deck needed

### LINKED ISSUE #3352: Space Invaders Won't Create Tokens from hand if Deck is empty
In bellow example I play space invaders with 3 creatures in hand and no cards in deck. The creatures in hand are not selectable 

image

### PR DESCRIPTION
**Pull request by strib**
Fixes #3352

### CODE REVIEW

File: server/game/cards/06-WoE/SpaceInvaders.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ class SpaceInvaders extends Card {
                     target: context.player,
                     amount: context.target.length,
                     cards: context.target,
-                    cardLocation: 'hand'
+                    cardLocation: 'hand',
+                    alwaysSucceed: true
```
Comment (romanogit) [inline server/game/cards/06-WoE/SpaceInvaders.js:20]: I think something is wrong here.

Comment (strib) [reply server/game/cards/06-WoE/SpaceInvaders.js:20]: `MakeTokenCreature` checks that the deck is not empty, and that check isn't needed for this card.  `alwaysSucceed` was the easiest way to get around that but I'm open to other suggestions for sure.

Comment (romanogit) [reply server/game/cards/06-WoE/SpaceInvaders.js:20]: I believe it is the same root cause of https://github.com/keyteki/keyteki/issues/1840

Comment (romanogit) [reply server/game/cards/06-WoE/SpaceInvaders.js:20]: disregard the previous comment, I was thinking about TheVisibleHand.
This one probably needs a different GameAction. Let me think about it

**Comment by romanogit** [comment]
Used your test case in https://github.com/keyteki/keyteki/pull/3363

**Pull request closed without merging**